packages/hooks/src/classes/FlatfileRecord.ts

```diff
@@ -229,6 +229,13 @@ export class FlatfileRecord<
     return null
   }
 
+  /**
+   * Returns an array of the error-level messages for the record
+   */
+  public getErrors(): IRecordInfo<M>[] {
```

Comment:
Could we add an optional parameter to pass the key here?

Reply:
I have added a `fields` argument which can take one `keyof M` or an array of `keyof M`.